src/main/java/bio/terra/service/snapshotbuilder/query/filtervariable/BinaryFilterVariable.java

```diff
@@ -12,10 +13,10 @@ public record BinaryFilterVariable(
   private static final String SUBSTITUTION_TEMPLATE = "<fieldVariable> <operator> <value>";
 
   @Override
-  public String renderSQL() {
+  public String renderSQL(CloudPlatformWrapper platform) {
     return new ST(SUBSTITUTION_TEMPLATE)
-        .add("operator", operator.renderSQL())
-        .add("value", value.renderSQL())
+        .add("operator", operator.renderSQL(platform))
+        .add("value", value.renderSQL(platform))
         .add("fieldVariable", fieldVariable.renderSqlForWhere())
```

Comment:
That's a good question! I hadn't thought about it but now I am kinda thinking at least for now, I would rather hold off on making that change. Since it doesn't really need platform to begin with. But maybe it would be better to add it for completeness's sake. So I'm not sure, what do you think?

Reply:
I'm okay holding off, we can always add it if we need it later.